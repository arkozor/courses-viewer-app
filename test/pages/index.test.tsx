import React from 'react'
import { render } from '../testUtils'
import HomePage from '../../pages'

describe('CoursesViewerApp page', () => {
    it('matches snapshot', () => {
        const { asFragment } = render(<HomePage />, {})
        expect(asFragment()).toMatchSnapshot()
    })

    // it('clicking button triggers alerts', () => {
    //     const { getByText } = render(<HomePage />, {})
    //     window.alert = jest.fn()
    //     fireEvent.click(getByText('Test Button'))
    //     expect(window.alert).toHaveBeenCalledWith('With typescript and Jest')
    // })
})
