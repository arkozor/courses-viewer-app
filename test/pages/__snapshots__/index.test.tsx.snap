// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CoursesViewerApp page matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <div>
      <div
        class="MuiButtonBase-root MuiChip-root cSharpChip chip MuiChip-clickable"
        role="button"
        tabindex="0"
      >
        <span
          class="MuiChip-label"
        >
          CSharp
        </span>
        <span
          class="MuiTouchRipple-root"
        />
      </div>
      <div
        class="MuiButtonBase-root MuiChip-root javaChip chip MuiChip-clickable"
        role="button"
        tabindex="0"
      >
        <span
          class="MuiChip-label"
        >
          Java
        </span>
        <span
          class="MuiTouchRipple-root"
        />
      </div>
      <div
        class="MuiButtonBase-root MuiChip-root jsChip chip MuiChip-clickable"
        role="button"
        tabindex="0"
      >
        <span
          class="MuiChip-label"
        >
          JavaScript
        </span>
        <span
          class="MuiTouchRipple-root"
        />
      </div>
      <div
        class="MuiButtonBase-root MuiChip-root phpChip chip MuiChip-clickable"
        role="button"
        tabindex="0"
      >
        <span
          class="MuiChip-label"
        >
          Php
        </span>
        <span
          class="MuiTouchRipple-root"
        />
      </div>
    </div>
    <div>
      Hello World
    </div>
  </div>
</DocumentFragment>
`;
